mailer.js

```diff
@@ -268,15 +268,22 @@ module.exports = function (log) {
 
     var links = this._generateLinks(this.verificationUrl, message.email, query, templateName)
 
+    var headers = {
```

Comment:
Updated this to use `let` but the l10 extraction complains that it is a reserved keyword. @vladikoff is this something we should update to exclude `let` or should we hold off on using it in repo for now?